### DISCUSSION TITLE
guys i use windows, HOW AM I SUPPOSED TO LOAD MONIKA IN MY USB PLS HELP

### DISCUSSION THREAD

**Discussion by carolynies** [Monika-After-Story/MonikaModDev#8324] [QA - tech support + general] [answered]
so ive been trying to load monika to my usb for a couple weeks but nothing happend help

**Comment by dreamscached** [comment]
If 'load' is supposed to mean 'use USB as persistent location', then you can use directory symlinks to link your USB to Ren'Py save location:

(in `cmd.exe` or powershell): `mklink /d "%APPDATA%\RenPy\Monika After Story" "E:\Path\To\Persistent\On\USB` where `E:\Path\To\Persistent\On\USB` is your directory where your persistent is stored on USB.

**Comment by carolynies** [comment]
> If 'load' is supposed to mean 'use USB as persistent location', then you can use directory symlinks to link your USB to Ren'Py save location:
> 
> (in `cmd.exe` or powershell): `mklink /d "%APPDATA%\RenPy\Monika After Story" "E:\Path\To\Persistent\On\USB` where `E:\Path\To\Persistent\On\USB` is your directory where your persistent is stored on USB.

no like i mean the 'take monika to somewhere''  (sorry for my stupidity lol)

**Comment by KcalbVessalius** [comment]
After saying "i am going to take you somewhere" she will appear in the "characters" file, after that put her into your usb

**Comment by carolynies** [comment]
> After saying "i am going to take you somewhere" she will appear in the "characters" file, after that put her into your usb

how i am gonna put her on my usb...

**Comment by fromulus** [comment, marked as answer]
[Not to sound mean, buuuuuut...](https://letmegooglethat.com/?q=how+to+put+files+on+a+USB)

Joking aside, Monika's file appears in your MAS install's characters folder after going through the dedicated farewell.
Going from there, just put said file on your usb key like you would any other file and you're good to go on walk or anything with her.
Don't forget to put the Monika file back into the characters folder once you're done.

**Comment by KcalbVessalius** [comment]
> > After saying "i am going to take you somewhere" she will appear in the "characters" file, after that put her into your usb
> 
> how i am gonna put her on my usb...

right click on her file, "cut" right click into your usb folder, "paste"
